src/services/expenseService.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const { models } = require('../models/models');
+
+class ExpenseService {
+  async createExpense(userId, spentAt, title, amount, category, note) {
+    try {
+      const newExpense = await models.Expense.create({
+        userId,
+        spentAt,
+        title,
+        amount,
+        category,
+        note,
+      });
+
+      return newExpense;
+    } catch (error) {
+      throw error;
```

Comment:
If what you catch does is only throwing the error, you might get rid of the whole `try/catch`